compiler/rustc_target/src/spec/mod.rs

```diff
@@ -1172,6 +1172,7 @@ supported_targets! {
     ("armv7-linux-androideabi", armv7_linux_androideabi),
     ("thumbv7neon-linux-androideabi", thumbv7neon_linux_androideabi),
     ("aarch64-linux-android", aarch64_linux_android),
+    ("riscv64-linux-android", riscv64_linux_android),
```

Comment:
It doesn't sound like it's required based on the [Target Tier Policy](https://doc.rust-lang.org/nightly/rustc/target-tier-policy.html#adding-a-new-target), but a lot of times adding a new target comes with a [Major Change Proposal](https://forge.rust-lang.org/compiler/mcp.html).

Reply:
Issue created [here](https://github.com/rust-lang/compiler-team/issues/640).